.gitignore

```diff
@@ -19,6 +19,8 @@ node_modules
 !tests/**/node_modules
 npm-debug.log
 sftp-config.json
+.cache-eslint-remote-tester
```

Comment:
any reason this cache isn't kept inside of `node_modules`? that way it gets blown away (as expected) with `npm ci`, and gitignored/eslintignored automatically

Reply:
I think the cache was initially hidden under `node_modules`. However as the cache can grow into tens of GBs it is often needed to be manually cleared in order to get some disk space for local development machines. Having the 10-20GB cache hidden under `node_modules` didn't provide the best _transparency_. Now that it's clearly visible it's easier to remove - in my opinion.

I'll reconsider moving it under `node_modules` but I don't really like that idea.

Since this repository will only run the tool in CI it is not mandatory to add this line to `.gitignore`. But I would recommend to include it just in case.

Reply:
i agree it should be in gitignore, but also it seems like it should be automatically pruned by the tool without the need for manual removal?

Reply:
Issue created https://github.com/AriPerkkio/eslint-remote-tester/issues/100. Thanks for the improvement idea. 👍 

The current location of the cache does not block this PR.

Reply:
Cache is now located at `node_modules/eslint-remote-tester/.cache-eslint-remote-tester`. Removed this line from `.gitignore`.